src/generated/resources/data/gtceu/tags/items/fluid_regulators.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_fluid_regulator",
+    "gtceu:mv_fluid_regulator",
+    "gtceu:hv_fluid_regulator",
+    "gtceu:ev_fluid_regulator",
+    "gtceu:iv_fluid_regulator",
+    "gtceu:luv_fluid_regulator",
+    "gtceu:zpm_fluid_regulator",
+    "gtceu:uv_fluid_regulator",
+    "gtceu:uhv_fluid_regulator",
+    "gtceu:uev_fluid_regulator",
+    "gtceu:uiv_fluid_regulator",
+    "gtceu:uxv_fluid_regulator",
+    "gtceu:opv_fluid_regulator"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.